en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -524,13 +524,14 @@ $players = Player::get()->exclude([
 ]);
 ```
 
-### `subtract`
+### `filterByList` and `excludeByList`
 
-You can subtract entries from a [DataList](api:SilverStripe\ORM\DataList) by passing in another DataList to `subtract()`
+You can exclude entries or include only entries in a [`DataList`](api:SilverStripe\ORM\DataList) based on the results of another list a by passing the other list to `excludeByList()` or `filterByList()` respectively.
```

Comment:
```suggestion
You can filter or exclude entries in a [`DataList`](api:SilverStripe\ORM\DataList) based on the contents of another list passed in to `filterByList()` or `excludeByList()` respectively.
```
Having filter first and exclude second is probably how most people think about this pair of methods

You can also do it in an EagerLoadedList ... not sure if mentioning that here would just be confusing though